src/YPushButton.h

```diff
@@ -216,6 +216,13 @@ class YPushButton : public YWidget
 	{ setLabel( str ); }
 
 
+    /**
+     * Activate the button. Can be used in tests to simulate user input.
+     *
+     * Derived classes are required to implement this.
+     **/
+    virtual void activate() = 0;
```

Comment:
The new added methods is will break the ABI, we will need to update the library SO version and also update the Qt and Ncurses plugins.

Reply:
We need changes in qt and ncurses anyway, so there is not much sense in merging this change alone. I will bump versions accordingly.